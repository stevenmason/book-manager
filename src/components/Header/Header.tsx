import React from "react";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";

const Header: React.FunctionComponent<{}> = () => {
  return (
    <AppBar position="static" color="default">
      <Toolbar>
        <Typography variant="h6" color="inherit">
          Book Manager
        </Typography>
      </Toolbar>
    </AppBar>
  );
};

export default Header;
