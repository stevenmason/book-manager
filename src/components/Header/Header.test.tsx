import React from "react";
import { render } from "@testing-library/react";

import Header from "./Header";

const PROPS = {};

it("renders correctly", () => {
  const { container } = render(<Header {...PROPS} />);
  expect(container.firstChild).toMatchSnapshot();
});
