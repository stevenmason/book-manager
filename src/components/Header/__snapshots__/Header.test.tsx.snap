// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorDefault MuiPaper-elevation4"
>
  <div
    class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <h6
      class="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit"
    >
      Book Manager
    </h6>
  </div>
</header>
`;
